internal/commands/controller/command.go

```diff
@@ -161,24 +162,20 @@ func (c *Command) Run(args []string) (ret int) {
 		return c.Error("timeout waiting for certs to be written", err)
 	}
 
-	// replace this with the Consul store
-	store := memory.NewStore(memory.StoreConfig{
-		Adapter: consulAdapters.NewSyncAdapter(logger.Named("consul-adapter"), client),
-		Logger:  logger.Named("state"),
-	})
-	group.Go(func() error {
-		store.SyncAtInterval(groupCtx)
-		return nil
+	store := store.New(store.Config{
+		Adapter:       consulAdapters.NewSyncAdapter(logger.Named("consul-adapter"), client),
+		Backend:       store.NewMemoryBackend(),
+		Binder:        nil, // TODO VM runtime implementation
+		Logger:        logger.Named("store"),
+		Marshaler:     nil, // TODO VM runtime implementation
+		StatusUpdater: nil, // TODO VM runtime implementation
```

Comment:
These TODOs will not be part of the refactor as this command is new and purely for VM support